### ISSUE TITLE
New Specs: odbc_ini and odbcinst_ini

### ISSUE THREAD

**Issue by xiangce** [RedHatInsights/insights-frontend-assets#31]
https://github.com/RedHatInsights/insights-core/pull/1128
~~~
    odbc_ini = RegistryPoint()
    odbcinst_ini = RegistryPoint()
    odbc_ini = simple_file("/etc/odbc.ini")
    odbcinst_ini = simple_file("/etc/odbcinst.ini"


filter_list = [
        '[',
        'DRIVER', 'Driver', 'driver',
        'SERVER', 'Server', 'server',
        'NO_SSPS', 'No_ssps', 'no_ssps',
]
add_filter(Specs.odbc_ini, filter_list)
~~~

**Comment by kylape** [comment]
Resolved in 3fbbf607085e1e77e8e0a9f26881c8c74aedfef5

**Comment by xiangce** [comment]
Thanks.
